test/controllers/ConfirmTransferControllerSpec.scala

```diff
@@ -1,34 +1,33 @@
 package controllers
 
-import java.util.UUID
 import com.github.tomakehurst.wiremock.WireMockServer
-import com.github.tomakehurst.wiremock.client.WireMock.{equalToJson, ok, okJson, post, serverError, urlEqualTo}
+import com.github.tomakehurst.wiremock.client.WireMock.{status => _, _}
```

Comment:
Perhaps there's a reason but could we not just import as
`import com.github.tomakehurst.wiremock.client.WireMock._`?
or leave it explicit like before so we know what we're using?

Reply:
Yeah, `._` makes sense. The thing you're referring to is something that IntelliJ did when I selected "Optimize imports"; I should probably not rely on it too much especially since it's not very good at handling the imports of Implicits.